.github/workflows/cd-dashboard.yml

```diff
@@ -11,10 +11,7 @@ env:
   PROJECT_ID: ${{ secrets.GCP_PROJECT }}
   GCP_SA_KEY: ${{ secrets.GCP_SA_KEY }}
   GCP_RUNNER_SA: ${{ secrets.GCP_SA_RUNNER }}
-  NEXT_PUBLIC_SIDETREE_METHOD: ${{ secrets.NEXT_PUBLIC_SIDETREE_METHOD }}
-  NEXT_PUBLIC_OPERATOR: ${{ secrets.NEXT_PUBLIC_OPERATOR }}
-  NEXT_PUBLIC_METHOD: ${{ secrets.NEXT_PUBLIC_METHOD }}
-  NEXT_PUBLIC_DESCRIPTION: ${{ secrets.NEXT_PUBLIC_DESCRIPTION }}
+  SIDETREE_METHOD: ${{ secrets.SIDETREE_METHOD }}
```

Comment:
This should be generalized, consider the future case where the same github secrets are used to deploy multiple methods to multiple testnets.

Reply:
specifically `secrets.SIDETREE_METHOD` -> `secrets.DID_METHOD_ELEM_TESTNET` -> `elem:ropsten` or something to that effect... CD is for a specific env... this might not even be a secret.

Reply:
also considering renaming the file to something more specific than `cd-dashboard.yml`